yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
delete yarn.lock from root dir

Reply:
oooops, it seems I wanna type yarn install somewhere else, but lose focus on another terminal.